core/app/modules/import-export/assets/js/pages/import/import-resolver/components/conflict/conflict.js

```diff
@@ -1,30 +1,39 @@
 import { useContext } from 'react';
 
 import { ImportContext } from '../../../../../context/import-context/import-context-provider';
+import { SharedContext } from '../../../../../context/shared-context/shared-context-provider';
 
 import ConflictCheckbox from './components/conflict-checkbox/conflict-checkbox';
 import Heading from 'elementor-app/ui/atoms/heading';
 import Text from 'elementor-app/ui/atoms/text';
 import Grid from 'elementor-app/ui/grid/grid';
 import Button from 'elementor-app/ui/molecules/button';
+import { eventTrackingDispatch } from 'elementor-app/event-track/events';
 
 export default function Conflict( props ) {
 	const importContext = useContext( ImportContext ),
+		sharedContext = useContext( SharedContext ),
 		manifest = importContext.data.uploadedData?.manifest,
+		{ referrer, wizardStepNum } = sharedContext.data,
 		getConflictTitle = ( id ) => {
 			const templateType = manifest.templates[ id ].doc_type,
 				summaryTitle = elementorAppConfig[ 'import-export' ].summaryTitles.templates?.[ templateType ];
 
 			return summaryTitle?.single || templateType;
 		},
-		getEditTemplateButton = ( editUrl ) => (
+		getEditTemplateButton = ( editUrl, title ) => (
 			<Button
 				className="e-app-import-resolver-conflicts__edit-template"
 				url={ editUrl }
 				target="_blank"
 				icon="eicon-editor-external-link"
 				text={ __( 'Edit Template', 'elementor' ) }
 				hideText
+				onClick={ () => {
+					if ( 'kit-library' === referrer && props.viewConflictItemEvent ) {
+						props.viewConflictItemEvent( title );
```

Comment:
Same comment as other places.

Reply:
Also, don't give away what you do in the callback by calling it like that.
Make the event name generic.

Reply:
> Also, don't give away what you do in the callback by calling it like that.
> Make the event name generic.

Changed to `onClick`